packages/ui/src/1_atoms/Icon/iconSvgPaths.ts

```diff
@@ -73,4 +73,10 @@ export const IconSvgPaths: Record<IconName, string[]> = {
   'x-mark': [
     'M24 2.41714L21.5829 0L12 9.58286L2.41714 0L0 2.41714L9.58286 12L0 21.5829L2.41714 24L12 14.4171L21.5829 24L24 21.5829L14.4171 12L24 2.41714Z',
   ],
+  copy: [
+    'M15.4737 0H2.21053C0.994737 0 0 0.940909 0 2.09091V16.7273H2.21053V2.09091H15.4737V0ZM18.7895 4.18182H6.63158C5.41579 4.18182 4.42105 5.12273 4.42105 6.27273V20.9091C4.42105 22.0591 5.41579 23 6.63158 23H18.7895C20.0053 23 21 22.0591 21 20.9091V6.27273C21 5.12273 20.0053 4.18182 18.7895 4.18182ZM18.7895 20.9091H6.63158V6.27273H18.7895V20.9091Z',
+  ],
+  exit: [
+    'M12 3.11111L10.872 4.20778L12.936 6.22222H4.8V7.77778H12.936L10.872 9.78444L12 10.8889L16 7L12 3.11111ZM1.6 1.55556H8V0H1.6C0.72 0 0 0.7 0 1.55556V12.4444C0 13.3 0.72 14 1.6 14H8V12.4444H1.6V1.55556Z',
```

Comment:
```suggestion
    'M18 4.88889L16.308 6.61222L19.404 9.77778H7.2V12.2222H19.404L16.308 15.3756L18 17.1111L24 11L18 4.88889ZM2.4 2.44444H12V0H2.4C1.08 0 0 1.1 0 2.44444V19.5556C0 20.9 1.08 22 2.4 22H12V19.5556H2.4V2.44444Z',
```
updated exit icon with 0 0 24 24 viewbox by default, 
and I'm not sure about changes in the icon component regarding iconProps, because in MenuItem we won't use icons with different sizes.
As I understand it, you are importing iconProps properties to change the viewBox and the size of the exit icon, 
we have a default size of 16px and the viewBox is 0 0 24 24, so if you just replace the exit icon with the default size, it will be the same as all others and the copy icon as well and you don't need to import iconProps